content/en/docs/reference/glossary/iop.md

```diff
@@ -0,0 +1,7 @@
+---
+title: IOP
+test: n/a
+---
+
+IOP, short for Istio Operator, is the custom resource used to configure
+an Istio installation when installing using the `istioctl install` command.
```

Comment:
Not only for standalone operator(`istioctl install`) but also for in-cluster operator, they share the same IOP resource.